qa-pipelines/tasks/lib/azure-aks.sh

```diff
@@ -46,22 +46,41 @@ azure_dns_clear() {
             --name ${matching_record_set}
     done
 }
+lbs_info() {
+    kubectl get svc -n $namespace -o json |
```

Comment:
namespace should be set in the function, this is only going to work if it's called from a place where the `namespace` variable is already set